### ISSUE TITLE
Notebook page should open to last viewed spot

### ISSUE THREAD

**Issue by nicolasmroberts** [StraboSpot/StraboSpot-Mobile#58]
If no spot is selected in the map, then the last viewed spot opens when you tap the notebook button.
![IMG_2380](https://user-images.githubusercontent.com/35352527/84044290-fa5a3780-a96c-11ea-8760-00bd6c89901c.jpeg)

**Comment by sdamaraju** [comment]
383a46a4d2c5c953e5823d8620290d417ec1ff73

**Comment by jessicagood** [comment]
@nicolasmroberts If we have a Spot open in the Notebook panel it is the current selected Spot and thus shows as selected on the map. @sdamaraju has implemented this request properly but Doug and the students don't like how now a Spot is always selected on the map - you cannot deselect a Spot since there is always a selected Spot (the most recent Spot if no other Spot has been selected). My suggestion would be to put it back how it was (where you can not have a Spot selected) but if there is no Spot selected and you open the notebook panel then show a list of the most recent Spots (which would be the same as the  same as Recent Views in the Spots List in the Main Menu) so we already have the data to show this pretty easily. What do you think?

**Comment by sdamaraju** [comment]
@jessicagood , Initially, technically I tried to decouple the props.selectedspot and notebook panel, to use a different way to use the most recent selected spot for the notebook panel to display the details.

Unfortunately, there is a huge dependency for the notebook panel on the props.selectedspot, I had to change at least 10 files (still incomplete fix), with too many changes to get it to work and still it breaks at a lot of places, (Note that, notebook panel includes, measurements, compass modal details and many other sub components, which all are dependent on props.selectedspot), even after so many changes, there were issues, 
Hence I had to revert to the logic where, a spot is always stays selected with the most recent spot, when no spot is actually selected.

We can revert this logic and implement a recent views display here as suggested by you. 
@nicolasmroberts waiting for your confirmation
